src/components/layout/header/Header.module.scss

```
@@ -0,0 +1,70 @@
+.header {
+  @include flexbox;
+
+  position: relative;
+
+  align-items: center;
+  justify-content: space-between;
+
+  max-width: 144rem;
+  padding: 1rem 4rem;
+
+  background: $opacity-white-10;
+  backdrop-filter: $header-blur;
+  border-radius: 99.9rem;
+
+  &-logo {
+    @include text-style-quantico(32);
+
+    line-height: 4rem;
+
+    &-mobile-size {
+      @include pos-center-x;
+    }
+  }
+
+  &-outer-container {
+    @include flexbox($gap: 3.2rem);
+
+    @media (max-width: $tablet-breakpoint) {
+      gap: 0.8rem;
+    }
+  }
+
+  &-inner-container {
+    @include flexbox($gap: 0.8rem);
+  }
```

```suggestion
  &-container {
    &-outer {
      // ...
    }
    &-inner {
      // ...
    }
  }
```

요것도 중요하진 않지만, 클래스명을 `logo-container-outer`, `logo-container-inner` 처럼 한다면 위와 같이 중첩문으로 쓸수 있을거 같습니다!

---

반영했습니다!